.github/workflows/ci-release.yml

```diff
@@ -43,7 +43,7 @@ jobs:
           # Build arguments to feed to the single gradlew publish command
           if [ "${{ matrix.java }}" = "8" ]; then
             FIRST_GRADLE_TARGETS=" clean check"
-            SECOND_GRADLE_TARGETS=" publish"
+            SECOND_GRADLE_TARGETS=" --no-parallel publish"
```

Comment:
I think you also need to remove the parallel flag from `gradle.properties` file

Reply:
or does the explicit flag override it?

Reply:
The explicit command line flag takes precedence. I checked with local build.